src/database/queries/baseFieldLocalizations/createOrUpdateByPrimaryKey.sql

```diff
@@ -13,5 +13,4 @@ ON CONFLICT (base_field_id, language)
 DO UPDATE SET
   label = EXCLUDED.label,
   description = EXCLUDED.description
-WHERE EXCLUDED.base_field_id = :baseFieldId AND EXCLUDED.language = :language
```

Comment:
Good catch, @slifty!

For further context, I believe this is the conversation you were referring to in the commit message: https://github.com/PhilanthropyDataCommons/service/pull/1123#discussion_r1718929284

Reply:
Ah yes good find -- I'll link to that directly.